src/main/java/edu/ucla/library/bucketeer/MainVerticle.java

```diff
@@ -45,6 +46,20 @@ public void start(final Future<Void> aFuture) {
                         // Next, we associate handlers with routes from our specification
                         routerFactory.addHandlerByOperationId(Op.GET_PING, new GetPingHandler());
                         routerFactory.addHandlerByOperationId(Op.LOAD_IMAGE, new LoadImageHandler());
+
```

Comment:
Would you break this below out into its own handler and put it in the handlers package? Something like LoadImageFailureHandler. This will keep this MainVerticle file simpler/cleaner.